lib/crypto/c_src/crypto.c

```diff
@@ -2331,6 +2333,27 @@ static ERL_NIF_TERM bin_from_bn(ErlNifEnv* env, const BIGNUM *bn)
     return term;
 }
 
+static ERL_NIF_TERM strong_rand_range_nif(ErlNifEnv* env, int argc, const ERL_NIF_TERM argv[])
+{/* (Range) */
+    BIGNUM *bn_range, *bn_rand;
+    ERL_NIF_TERM ret;
+
+    if(!get_bn_from_bin(env, argv[0], &bn_range)) {
+        return enif_make_badarg(env);
+    }
+
+    bn_rand = BN_new();
+    if (BN_rand_range(bn_rand, bn_range) != 1) {
```

Comment:
Mmmmh, I considered that (as it's a very common pattern), but the [documentation](https://wiki.openssl.org/index.php/Manual:BN_rand(3)) explicitly states that either '0' or '1' shall be returned for failure or success, respectively. 
It would still work after that change, but I worry whether it could suddenly behave unexpectedly if, let's say, one day it starts returning '2' of '0xBEEF' to signal something else entirely?